apps/cli/src/args.ts

```diff
@@ -31,12 +33,11 @@ export const CLI_ARGS_OPTIONS = {
   },
 };
 
-export const HELP = `Usage:
-    $ webstudio commands [flags...]
-  Commands:
-    link <shared link>              Login to Webstudio with shared link
-
-    Flags:
-    --help, -h                      Show this help message
-    --version, -v                   Show the version of this script
+export const HELP = `\n Usage:
+\t $ webstudio commands [flags...] \n
+\t Commands:
+\t\t link <shared link> \t Login to Webstudio with shared link \n
+\t Flags:
+\t\t --help, -h \t\t Show this help message
+\t\t --version, -v \t\t Show the version of this script
```

Comment:
what was wrong with previous output?

Reply:
@kof suggested to switch to this on the previous PR
https://github.com/webstudio-is/webstudio-builder/pull/2018#discussion_r1281579669

Reply:
But it is like same but with explicit space characters. Wouldn't say it is nicer to read. I'd keep it same as before.

Adding dependency just for this case is not good.

Reply:
oh this looks like a mess, its not how I meant it the idea is to use stripIndent to actually make it look nice:
```
stripIndent(`
  Usage:
  ...
`)

Reply:
I wouldn't say this gives too much value. Top level help template is good as is. Error messages are fine with separate strings. Adding dependency would not benefit end user.

Reply:
Should i switch back to the old way ?

Reply:
@TrySound there is a lot of ugly indentation going on when using plain template strings, but JK did the wrong thing, so now it looks even worse, the idea  with stripIndent is to be able to format everything just like you would normaly and this is not perf-sensitive case, dependency is tiny ... once you see how it has to look like, you will never want to go back to vanilla template strings for large blocks of text

Reply:
I used a similiar function for tests in JSS, without it, those strings would be ugly as hell

https://github.com/cssinjs/jss/blob/master/packages/jss/tests/integration/rules.js#L111